appserver/payara-appserver-modules/notification-xmpp-core/src/main/java/fish/payara/notification/xmpp/XmppNotificationConfigurer.java

```diff
@@ -84,7 +84,7 @@
     @Param(name = "securityDisabled", defaultValue = "false", optional = true)
     private Boolean securityDisabled;
 
-    @Param(name = "roomName")
+    @Param(name = "roomID")
     private String roomName;
```

Comment:
Bleurgh! Refactoring just the parameter name isn't very nice! 😝 
It's not just messy - as I mentioned in the previous comment, the property returned by the get command is separate to this parameter name, so you still have some refactoring to do!

A quick grep (and off the top of my head) you'll need to refactor all occurrences of _roomName_ to _roomID_:
* In this class
* In the XmppNotifierConfiguration class
* In the GetXmppNotifierConfiguration class
* In the XmppNotifierConfigurationExecutionOptions class